composer.json

```diff
@@ -26,11 +25,21 @@
   "autoload": {
     "psr-0": { "Ambta\\DoctrineEncryptBundle": "" }
   },
+  "autoload-dev": {
+    "psr-4": {
+      "Doctrine\\Tests\\": "vendor/doctrine/orm/tests/Doctrine/Tests"
+    }
+  },
   "target-dir": "Ambta/DoctrineEncryptBundle",
   "authors": [
     {
       "name": "GiveMeAllYourCats",
       "email": "specamps@gmail.com"
     }
-  ]
+  ],
+  "config": {
+      "preferred-install": {
+          "doctrine/orm": "source"
+      }
+  }
```

Comment:
The changes here were added to be able to run the functional-tests.  `Autoload-dev` so that the code is found, `preferred-install` from source so that the tests will be installed.  Normally the dist-version of the library is installed, which most of the time do not include the tests.